intdataplane/int_dataplane.go

```diff
@@ -666,32 +667,64 @@ func (d *InternalDataplane) apply() {
 
 	// Next, create/update IP sets.  We defer deletions of IP sets until after we update
 	// iptables.
-	for _, w := range d.ipSets {
-		w.ApplyUpdates()
+	var ipSetsWG sync.WaitGroup
+	for _, ipSets := range d.ipSets {
+		ipSetsWG.Add(1)
+		go func(ipSets *ipsets.IPSets) {
+			ipSets.ApplyUpdates()
+			ipSetsWG.Done()
+		}(ipSets)
```

Comment:
Out of interest, is passing ipSets through as an arg, as here, equivalent to adding `ipSets := ipSets` at the start of the loop?  Or are there other factors?